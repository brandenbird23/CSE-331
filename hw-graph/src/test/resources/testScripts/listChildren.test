# listChildren test file
# tests if listChildren can be added to graph

# create graph
CreateGraph graph1

# create node
AddNode graph1 node1

# create edge
AddEdge graph1 node1 node1 edge1

# print the nodes in graph
ListNodes graph1
ListChildren graph1 node1