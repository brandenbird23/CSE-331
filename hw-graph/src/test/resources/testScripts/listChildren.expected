# listChildren test file
# tests if listChildren can be added to graph

# create graph
created graph graph1

# create node
added node node1 to graph1

# create edge
added edge edge1 from node1 to node1 in graph1

# Print the nodes in graph
graph1 contains: node1
the children of node1 in graph1 are: node1(edge1)